packages/components/src/index.ts

```diff
@@ -1,2 +1,59 @@
-// Example of exporting components to be consumed from main @kaizen/components
+// Components inside this package
 export * from "./KaizenProvider"
+
+// Re-exports which need to be moved into this package
+export * from "@kaizen/a11y"
+export * from "@kaizen/brand"
+export * from "@kaizen/brand-moment"
+export * from "@kaizen/button"
+export * from "@kaizen/component-base"
+export * from "@kaizen/date-picker"
+export * from "@kaizen/draft-avatar"
+export * from "@kaizen/draft-badge"
+export * from "@kaizen/draft-card"
+export * from "@kaizen/draft-collapsible"
+export * from "@kaizen/draft-divider"
+export * from "@kaizen/draft-empty-state"
+export * from "@kaizen/draft-filter-menu-button"
+export * from "@kaizen/draft-form"
+export * from "@kaizen/draft-guidance-block"
+export * from "@kaizen/draft-hero-card"
+export * from "@kaizen/draft-illustration"
+export * from "@kaizen/draft-likert-scale-legacy"
+export * from "@kaizen/draft-menu"
+export * from "@kaizen/draft-modal"
+export * from "@kaizen/draft-page-layout"
+export * from "@kaizen/draft-popover"
+export * from "@kaizen/draft-table"
+export * from "@kaizen/draft-tabs"
+export * from "@kaizen/draft-tag"
+export * from "@kaizen/draft-tile"
+export * from "@kaizen/draft-title-block-zen"
+export * from "@kaizen/draft-tooltip"
+export * from "@kaizen/draft-well"
+export * from "@kaizen/hosted-assets"
+export * from "@kaizen/loading-skeleton"
+export * from "@kaizen/loading-spinner"
+export * from "@kaizen/notification"
+export * from "@kaizen/pagination"
+export * from "@kaizen/progress-bar"
+export * from "@kaizen/responsive"
+export * from "@kaizen/rich-text-editor"
+export * from "@kaizen/split-button"
+
+// Special cases
+
+// Part of design tokens is already exported from KaizenProvider
+export {
+  makeCssVariableDefinitionsMap,
+  makeCSSVariableTheme,
```

Comment:
@gyfchong I don't think you got what it was that is killing me here ;)
`makeCssVar...` vs `makeCSSVar...` - I wish those were consistent.

Reply:
OH haha